soundata/annotations.py

```diff
@@ -2,6 +2,21 @@
 """
 import numpy as np
 
+#: Time units
+TIME_UNITS = {
+    "s": "seconds",
+    "ms": "miliseconds",
+}
+
+#: Confidence units
+CONFIDENCE_UNITS = {
+    "likelihood": "score between 0 and 1",
+    "binary": "0 or 1",
```

Comment:
Is it worth to have two different "units" depending if the confidence is derived from the estimation of a model or manually labeled?

Reply:
I'm not sure - a human annotator can use 1.0 to indicate they are confident in their annotations. Practically there's no different in how the values would be used versus a binary confidence type. Indicating how the dataset was annotated is more of a metadata thing, right? I think it could potentially be beneficial to enforce a consistent "model" for confidence (float in [0,1]) for both humans and machines.

Perhaps we can start without a unit for confidence (keeping it always as a float in [0, 1]), and only add it if we see there is a real need for it based on user feedback?